src/main/java/com/exactpro/th2/infraoperator/util/CustomResourceUtils.java

```diff
@@ -102,7 +100,7 @@ private static String concatFullName(String namespace, String name) {
         return namespace + "." + name;
     }
 
-    private static String hashNameIfNeeded(String resName) {
+    public static String hashNameIfNeeded(String resName) {
```

Comment:
question: Do we need it to be public? There is `extractHashedName` function that extracts name from a `Th2CustomResource` instance. Don't think we need to use `hashNameIfNeeded` anywhere directly

Reply:
hashNameIfNeeded accepts String instead of Th2CustomResource

Reply:
Can we use `Th2CustomResource` instead of `String` in integration test? Having this function here makes it much easier to make a mistake.